docs/getting-started-guides/coreos/bare_metal_calico.md

```diff
@@ -87,7 +87,37 @@ Move the edited `master-config.yaml` to your Kubernetes master machine.  The Cor
 sudo coreos-install -d /dev/sda -C stable -c master-config.yaml
 ```
 
-Once complete, eject the bootable ISO and restart the server.  When it comes back up, you should have SSH access as the `core` user using the public key provided in the `master-config.yaml` file.  It may take a few minutes for the machine to be fully configured with Kubernetes and Calico.
+Once complete, eject the bootable ISO and restart the server.  When it comes back up, you should have SSH access as the `core` user using the public key provided in the `master-config.yaml` file.
```

Comment:
Why not? The `coreos-install` will put CoreOS on `/dev/sda`. If the iso is still connected, you'll boot back into the pre-install state.